### PR TITLE
Fix getTarget docs

### PR DESCRIPTION
**Pull request by spacey-sooty**
Resolves https://github.com/PhotonVision/photonvision/issues/1403

### CODE REVIEW

File: photon-lib/py/photonlibpy/photonCamera.py [hunk 1]
```diff
@@ -116,7 +116,7 @@ def getLatestResult(self) -> PhotonPipelineResult:
         retVal = PhotonPipelineResult()
         packetWithTimestamp = self._rawBytesEntry.getAtomic()
         byteList = packetWithTimestamp.value
-        timestamp = packetWithTimestamp.time
+        packetWithTimestamp.time
```
Comment (mcm001) [inline photon-lib/py/photonlibpy/photonCamera.py:119]: Wot

**Review by mcm001 — APPROVED** [review]

**Review by mcm001 — CHANGES_REQUESTED** [review]
Wpiformat be tripping

File: photon-targeting/src/main/native/include/photon/targeting/PhotonPipelineResult.h [hunk 2]
```diff
@@ -120,6 +120,7 @@ class PhotonPipelineResult {
 
   /**
    * Returns a reference to the vector of targets.
+   * <p> Returned in the order set by target sort mode. </p>
    * @return A reference to the vector of targets.
```
Comment (mcm001) [inline photon-targeting/src/main/native/include/photon/targeting/PhotonPipelineResult.h:124]: Nit: this isn't really "a reference", that's not a thing in Java. Should probably be re-worded?

**Review by mcm001 — APPROVED** [review]

File: photon-targeting/src/main/native/include/photon/targeting/PhotonPipelineResult.h (hunk 2 above)
Comment (spacey-sooty) [reply photon-targeting/src/main/native/include/photon/targeting/PhotonPipelineResult.h:124]: The Java docs say its a copy which is probably more accurate

Comment (mcm001) [reply photon-targeting/src/main/native/include/photon/targeting/PhotonPipelineResult.h]: Ok sorry this is what I get for reviewing code on my phone. This is actually a reference, it's just a span. We should consider changing this to be a span of const targets, but that's out of scope. Can you put it back to the reference verbiage? And leave Java as copy?

Although Java is weird. It's a list of the same underlying target objects, so I suspect that modifying the ones in the returned list mutate the result too? whack

**Pull request merged**